src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -347,16 +373,14 @@ void AttitudeEstimatorQ::task_main()
 			continue;
 		}
 
-		update_parameters(false);
-
 		// Update sensors
-		sensor_combined_s sensors;
+		sensor_combined_s sensors = _sub_sensor.get();
```

Comment:
Does this do a copy every time? (Just trying to understand how the the interface works.)